index.js

```diff
@@ -76,6 +76,13 @@ module.exports = {
     this.jshintrc = app.options.jshintrc;
   },
 
+  postBuild: function () {
```

Comment:
I thought that but I was getting init, unless you are happy with me exiting after error?